### PR TITLE
 _1password-gui-beta: 8.8.0-165.BETA -> 8.8.0-215.BETA

### PR DESCRIPTION
**Pull request by conradev**
###### Description of changes

I updated the 1Password GUI Beta to the latest version, as well as added support for `aarch64-linux`. Support for `aarch64` won't be ready for `default` until 8.8.0 is released.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SebTM** [comment]
LGTM, we should adapt the non-beta version with this improvement as well and we could generalize the two files (because only minor changes). Also we should add an update-script according to the non-beta package (https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/1password-gui/update.sh)

File: pkgs/applications/misc/1password-gui/beta.nix [hunk 1]
```diff
@@ -44,12 +44,19 @@ let
...
-    url = "https://downloads.1password.com/linux/tar/beta/x86_64/1password-${version}.x64.tar.gz";
-    sha256 = "sha256-ZpKAkuIMeHEFdz/od/sKDh8VXoWOZYO8GjvMiho3D4A=";
-  };
+  src =
+    if stdenv.hostPlatform.system == "aarch64-linux" then
+      fetchurl {
+        url = "https://downloads.1password.com/linux/tar/beta/aarch64/1password-${version}.arm64.tar.gz";
+        sha256 = "sha256-LqZAewI6dU9lalVL6A/oPUopS8HZ09SZpRrurVq2wKI=";
+      }
+    else
```
Comment (supermarin) [inline pkgs/applications/misc/1password-gui/beta.nix:55]: while you're at it, maybe support macOS downloads as well?

Comment (maxeaubrey) [reply pkgs/applications/misc/1password-gui/beta.nix:55]: I don't think making this very Linux specific derivation mac compatible would be in scope for a version bump.

**Review by amaxine — COMMENTED** [review]
@conradev `215` came out yesterday, could you bump it to latest beta? I'm around to merge it now :)

@SebTM I agree, once 8.8.0 is out as stable we should do that. The update script however seems to rely on a repo that doesn't get updated, at least as far as I can tell?

**Comment by conradev** [comment]
@maxeaubrey done!

@supermarin I got it working for macOS, but I will move that to a separate PR, later (https://github.com/conradev/nixpkgs/commit/410c1ca0750dc10e8921a75d1b96f86302379f30)

I looked into consolidation, and the `discord` package has a pretty good system for managing cross-platform application binaries

**Comment by conradev** [comment]
@supermarin it looks like 1Password 8 refuses to be put anywhere outside of `/Applications` on macOS due to poor engineering. So, it's a no go for now.

**Comment by supermarin** [comment]
@conradev thanks for looking into it!

**Comment by SebTM** [comment]
> @supermarin it looks like 1Password 8 refuses to be put anywhere outside of `/Applications` on macOS due to poor engineering. So, it's a no go for now.

Maybe it's a good idea to point this out per mail or in the 1password.community they are likely to change things to support their user/OS projects as they also helped making the 1Password-Helper's on linux nixos-compatible and contributed to the nixos-module :)

**Pull request merged**